src/Compilers/Core/MSBuildTask/ManagedCompiler.cs

```diff
@@ -174,6 +174,12 @@ public ITaskItem[] References
             get { return (ITaskItem[])_store["References"]; }
         }
 
+        public bool ReportAnalyzer
```

Comment:
Hmm so did we already move the MSBuild task to Roslyn? I have been making changes in DEVDIV tree for a new switch that I added.